app/models/event_stream.rb

```diff
@@ -32,4 +32,48 @@ def emit_notifications
   rescue => err
     _log.log_backtrace(err)
   end
+
+  #
+  # Purging methods
+  #
+
+  def self.keep_ems_events
+    VMDB::Config.new("vmdb").config.fetch_path(:ems_events, :history, :keep_ems_events)
```

Comment:
Moving the key will require a data migration, so let's do that in a follow up PR.  I think the key should change as well to something like `events/history/keep_events`